docs/preview/04-Guidance/migrate-from-v1-to-v2.md

```diff
@@ -44,8 +48,10 @@ TemporaryBlobContainer.CreateIfNotExistsAsync(..., options =>
 });
 ```
 
+<!-- vale write-good.Passive = NO -->
 ### `TemporaryBlobFileOptions` ➡️ `TemporaryBlobContainerOptions`
-Previous versions had additional options on the `TemporaryBlobFile` test fixture to override/use Azure Blob files. This and the entire options on this fixture has been removed in v2 - as it is already implicitly available on the `TemporaryBlobContainerOptions`.
+Previous versions had additional options on the `TemporaryBlobFile` test fixture to override/use Azure Blob files. This and the entire options on this fixture is removed in v2 - because already implicitly available on the `TemporaryBlobContainerOptions`.
```

Comment:
```suggestion
Previous versions had options on the `TemporaryBlobFile` test fixture to override/use Azure Blob files. This and the entire options on this fixture is removed in v2 - because already implicitly available on the `TemporaryBlobContainerOptions`.
```